src/BaseInput.java

```diff
@@ -1,44 +1,56 @@
 import javax.swing.*;
 import java.io.BufferedReader;
-import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
 
 public class BaseInput {
 
-    public static String BaseReader(String id, String pass) {
-        System.out.println(id + pass);
-        boolean flag = false;
-        try{
-            BufferedReader baza = new BufferedReader(new FileReader("/Users/artemlaptev/Desktop/baza.txt"));
+    /**
+     * Reads user information from the database stored in the "baza.txt" file and validates
+     * the input ID and password. If a matching ID is found with the correct password,
+     * displays user information in a dialog box.
+     *
+     * @param inputId The input ID string to be validated.
+     * @param inputPass The input password string to be validated.
+     * @return {@code true} if the input ID and password are valid and match a record in the database,
+     *         {@code false} otherwise.
+     *
+     * @throws RuntimeException if an IOException occurs while reading the database file.
+     */
+    public static boolean readUserInputFromDatabase(String inputId, String inputPass) {
```

Comment:
it's always a bad idea to return "yes" / "no" strings to indicate that validation passed / failed, better use boolean for it